<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOrderDialog</class>
 <widget class="QDialog" name="EditOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Order</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Имя клиента</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="descriptionEdit">
       <property name="placeholderText">
        <string>Описание работы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phoneEdit">
       <property name="placeholderText">
        <string>Телефон клиента</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="costEdit">
       <property name="placeholderText">
        <string>Стоимость</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statusComboBox">
       <item>
        <property name="text">
         <string>Ожидается</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>В процессе</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Завершено</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Просрочено</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="startDateEdit">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="endDateEdit">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="photoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>Photo:</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadPhotoButton">
         <property name="text">
          <string>Load Photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Удалить заказ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
